src/test/streams_tests.cpp

```diff
@@ -9,8 +9,7 @@
 #include <boost/assign/std/vector.hpp> // for 'operator+=()'
 #include <boost/assert.hpp>
 #include <boost/test/unit_test.hpp>
-                    
-using namespace std;
+
 using namespace boost::assign; // bring 'operator+=()' into scope
```

Comment:
It is also possible to scope the `using boost::*` to the only function where it is used. But better keep it as is, when there are plans to move to cpp11 anyway.